config/webpack.config.commons.js

```diff
@@ -24,6 +24,7 @@ exports.addFlowBabelWebpackPlugin = function (webpackConfig) {
 
 exports.commonAliases = function () {
   return {
+    '@spice-project/spice-html5': `${paths.appSrc}/emptyFile.js`,
```

Comment:
Why is this necessary?  It would be nice to have a comment in the code to explain.